### ISSUE TITLE
map_launcher-0.1.3/ios/Classes/SwiftMapLauncherPlugin.swift:49:22: 'MKCoordinateRegionMake' is unavailable in Swift

### ISSUE THREAD

**Issue by Pebsie** [mattermoran/map_launcher#1]
Hi,

When trying to build an app using this plugin I'm getting this error in Xcode:
`/map_launcher-0.1.3/ios/Classes/SwiftMapLauncherPlugin.swift:49:22: 'MKCoordinateRegionMake' is unavailable in Swift`

The same error is there for `MKCoordinateSpanMake`. The error comes from the `launchMap` function.

Flutter doctor:
```
[✓] Flutter (Channel master, v1.10.15-pre.330, on Mac OS X 10.15 19A583, locale en-GB)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 3.3)
[✓] VS Code (version 1.39.1)
 
[✓] Connected device (4 available)            

• No issues found!

```

**Comment by mattermoran** [comment]
Hey. I think I've seen this before. I'll think a better solution later. As a temp workaround could you add these lines to your projects ios/Podfile
```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['ENABLE_BITCODE'] = 'NO'
      config.build_settings['SWIFT_VERSION'] = '4.2'
      if target.name == 'map_launcher'
        config.build_settings['SWIFT_VERSION'] = '4.0'
      end
    end
  end
end
```

Let me know if that works

**Comment by AdriEU** [comment]
I also have a problem running the app. I get:
- `map_launcher` does not specify a Swift version and none of the targets (`Runner`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.

**Comment by mattermoran** [comment]
@AdriEU  that's because this is a swift plugin and you have (most likely) objective-c flutter project. You can find info and workarounds for that here: https://github.com/flutter/flutter/issues/16049

**Comment by jczhong** [comment]
@mattermoran I got the same error and made the change as your comment. But it cause other plugin failed because they need 4.2.

**Comment by diegogarciar** [comment]
issue can be fixed just by changing line to 
`let region = MKCoordinateRegion(center: coordinate, span: MKCoordinateSpan(latitudeDelta: 0.01, longitudeDelta: 0.02))`

**Comment by diegogarciar** [comment]
#5

**Comment by mattermoran** [comment]
#5 fixes the issue. Should be working for both swift 4 and 5 now

**Comment by mattermoran** [comment]
Closing this issue as it's fixed in 0.3.2
